docs/tutorials/01_introductory/viz_picking.py

```diff
@@ -0,0 +1,159 @@
+# Build scene and add an actor with many objects.
+
+scene = window.Scene()
+
+label_actor = actor.label(text='Test')
+
+###############################################################################
+# This actor is made with 3 cubes of different orientation
+
+directions = np.array([[np.sqrt(2)/2, 0, np.sqrt(2)/2],
+                       [np.sqrt(2)/2, np.sqrt(2)/2, 0],
+                       [0, np.sqrt(2)/2, np.sqrt(2)/2]])
+fury_actor = actor.cube(centers, directions, colors, heights=radii)
+
+###############################################################################
+# Uncomment the following lines to see the same effect with spheres
+
+# fury_actor = actor.sphere(centers=centers,
+#                           colors=colors,
+#                           radii=radii)
+
+###############################################################################
+# Access the memory of the vertices of all the spheres
+
+vertices = utils.vertices_from_actor(fury_actor)
+num_vertices = vertices.shape[0]
+num_objects = centers.shape[0]
+
+###############################################################################
+# Access the memory of the colors of all the spheres
+
+vcolors = utils.colors_from_actor(fury_actor, 'colors')
+
+###############################################################################
+# Adding an actor showing the axes of the world coordinates
+ax = actor.axes(scale=(10, 10, 10))
+
+scene.add(fury_actor)
+scene.add(label_actor)
+scene.add(ax)
+scene.reset_camera()
+
+###############################################################################
+# Create the Picking manager
+
+pickm = pick.PickingManager()
+
+###############################################################################
+# Time to make the callback which will be called when we pick an object
+
+
+def left_click_callback(obj, event):
+
+    # Get the event position on display and pick
+
+    event_pos = pickm.event_position(showm.iren)
+    picked_info = pickm.pick(event_pos[0], event_pos[1],
+                             0, showm.scene)
+    print(picked_info)
+    vertex_index = picked_info['vertex']
+
+    # Calculate the objects index
+
+    object_index = np.int(np.floor((vertex_index / num_vertices) *
+                          num_objects))
+
+    # Find how many vertices correspond to each object
+    sec = np.int(num_vertices / num_objects)
+
+    if not selected[object_index]:
+        scale = 6/5
+        color_add = np.array([30, 30, 30], dtype='uint8')
+        selected[object_index] = True
+    else:
+        scale = 5/6
+        color_add = np.array([-30, -30, -30], dtype='uint8')
+        selected[object_index] = False
+
+    # Update vertices positions
+    vertices[object_index * sec: object_index * sec + sec] = scale * \
+        (vertices[object_index * sec: object_index * sec + sec] -
+         centers[object_index]) + centers[object_index]
+
+    # Update colors
+    vcolors[object_index * sec: object_index * sec + sec] += color_add
```

Comment:
same comment as above: `update_colors` function. Few people will understand this part of the tutorial